htdocs/core/class/html.form.class.php

```diff
@@ -6582,6 +6582,218 @@ public function select_duration($prefix, $iSecond = '', $disabled = 0, $typehour
 		return;
 	}
 
+	// phpcs:disable PEAR.NamingConventions.ValidFunctionName.ScopeNotCamelCaps
```

Comment:
For new inserted method, we must avoid this: Just rename your new method
select_ticket into
selectTicket
so you can remove the phpcs:disable and phpcs:enable tag

Reply:
ok thank you, it's fixed